index.html

```diff
@@ -4,7 +4,216 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/bootstrap-grid.min.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aleo&family=Josefin+Sans:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Bhromaon</title>
   </head>
-  <body></body>
+  <body>
+    <header class="header">
+      <div class="container container__header">
+        <h1 class="header__header">Bhromaon</h1>
+        <h3 class="header__subheader">Safe Travel Anywhere</h3>
+        <form action="/" method="get" class="header__search">
+          <div class="header__search__wrapper">
+            <input
+              type="search"
+              name="search"
+              class="header__search__search"
+              placeholder="Search Your Travel Destination"
+            />
+            <button type="button" class="header__search__button">
+              <i class="icon-search"></i> Search
+            </button>
+          </div>
+        </form>
```

Comment:
тут шось не так з БЕМом
```suggestion
        <form action="/" method="get" class="search header__search">
          <div class="search__wrapper">
            <input
              type="search"
              name="search"
              class="search__input"
              placeholder="Search Your Travel Destination"
            />
            <button type="button" class="search__button">
              <i class="icon-search"></i> Search
            </button>
          </div>
        </form>
```
якось так буде краще